projects/plugins/jetpack/tests/e2e/lib/env/prerequisites.js

```diff
@@ -81,6 +81,13 @@ async function buildPrerequisites( state ) {
 }
 
 export async function ensureConnectedState( requiredConnected = undefined ) {
+	if ( global.isLocalSite ) {
+		logger.prerequisites(
+			'Site is not local, skipping connection setup. Assuming required setup is already in place.'
+		);
+		return;
+	}
```

Comment:
I'm not particularly happy with repeated validation logic among most of the prerequisites functions. Don't think it's worth resolving, at least for now. In the future, when we'll have more prerequisites/validations we could rethink the approach.